oarepo_vocabularies/ui/theme/assets/semantic-ui/js/oarepo_vocabularies_ui/form/hooks.js

```diff
@@ -0,0 +1,68 @@
+import { useDepositApiClient } from "@js/oarepo_ui";
+import { useFormikContext } from "formik";
+import _isEmpty from "lodash/isEmpty";
+
+export const useVocabularyApiClient = (newChildItemParentId) => {
+  const { apiClient, createUrl } = useDepositApiClient();
+  // TODO: maybe - as useDepositApiClient is already calling formik, it could makse sense for it to pass
+  // entire formik object down to any users, however, this might be a bit tricky,
+  // because I destructure some properties in the useDepositApiClient, so I would have to return
+  // those explicitly from the useDepositApiClient which would become a bit cluttered
```

Comment:
I think this could be easily done, see: https://github.com/oarepo/oarepo-ui/pull/73#discussion_r1324306561. That way we won't need to re-use formik context here